packages/compass-e2e-tests/tests/in-use-encryption.test.ts

```diff
@@ -126,9 +135,11 @@ describe('FLE2', function () {
           '})'
       );
       await browser.clickVisible(Selectors.SidebarInstanceRefreshButton);
+      plainMongo = await MongoClient.connect(CONNECTION_STRING);
     });
 
     after(async function () {
+      await plainMongo.close();
```

Comment:
If the connection is established in `beforeEach`, this should probably run in an `afterEach` hook